.circleci/config.yml

```diff
@@ -584,12 +584,9 @@ jobs:
           name: forge version
           command: forge --version
       - run:
-          # The solc warnings check must be the first step to build the contracts, that way the
-          # warnings are output here. On subsequent runs, forge will read artifacts from the cache
-          # so warnings would not occur.
           name: solc warnings check
           command: |
-            forge build --deny-warnings || echo "export SOLC_WARNINGS_CHECK=1" >> "$BASH_ENV"
+            forge build --force --deny-warnings || echo "export SOLC_WARNINGS_CHECK=1" >> "$BASH_ENV"
```

Comment:
PR LGTM, but CI is failing because it still logs "Compiler run successful with warnings:" but there are no warnings logged. I have not seen this before but verified it also occurs locally
<img width="352" alt="image" src="https://github.com/user-attachments/assets/417b692e-fed9-4e7b-a102-fdd706f7ae20">

Reply:
This could be due to an interaction with the ignored warnings in the forge config

Reply:
Good guess, from my repro attempts that doesn't seem to be the cause though. Created https://github.com/foundry-rs/foundry/issues/8548 as a result

Reply:
Looks like https://github.com/foundry-rs/foundry/issues/8548 has been fixed in https://github.com/foundry-rs/compilers/pull/179. We need to update the foundry version to get the fix

Reply:
Yea was waiting to see when the foundry-rs/compilers bump would happen, just requested it in https://github.com/foundry-rs/foundry/issues/8548#issuecomment-2263480818